### DISCUSSION TITLE
How can I add ElementId attribute to Datagrid Filterable input fields?

### DISCUSSION THREAD

**Discussion by DurgaSanthi** [Megabit/Blazorise#2731] [General]
Hi,

I am using Blazorise DataGrid to display the data. On enabling the Filterable option, the input elements for filtering appears. Is there a way to add ElementId attribute instead of dynamic Id generation everytime?

These input controls seems to be dynamically rendered.

<input id="0HMAG49VHETI6" type="text" class="form-control" _bl_16="">

![image](https://user-images.githubusercontent.com/23058170/126969433-c6fce3bd-c5f1-4c47-ae8e-50784a7ba448.png)

**Comment by David-Moreira** [comment]
Hello,
Have you tried using the FilterTemplate so you can set your own input with your own configurations.
